django/db/migrations/questioner.py

```diff
@@ -114,13 +114,14 @@ def _ask_default(self, default=''):
         string) which will be shown to the user and used as the return value
         if the user doesn't provide any other input.
         """
-        print("Please enter the default value now, as valid Python")
+        print("Please enter the default value as valid Python.")
         if default:
             print(
-                "You can accept the default '{}' by pressing 'Enter' or you "
-                "can provide another value.".format(default)
+                "Accept the default '{}' by pressing 'Enter' or"
+                " provide another value.".format(default)
             )
-        print("The datetime and django.utils.timezone modules are available, so you can do e.g. timezone.now")
+        print("Datetime and django.utils.timezone modules are imported and it is possible "
```

Comment:
`Datetime` should probably continue being in lowercase since it references a package name?